lib/full-site-editing/block-templates.php

```diff
@@ -237,6 +280,15 @@ function gutenberg_get_block_templates( $query = array(), $template_type = 'wp_t
 		),
 	);
 
+	if ( 'wp_template_part' === $template_type && isset( $query['area'] ) ) {
```

Comment:
`'area'` is a new key under `$query` and isn't documented in the `gutenberg_get_block_templates` docblock. @Addison-Stavlo, could you follow up with that quick addition? 🙏

Reply:
Certainly!  My apologies for the oversight there earlier - https://github.com/WordPress/gutenberg/pull/32746

Reply:
Not at all, thank you!